### ISSUE TITLE
new version gives me aes compilation errors

### ISSUE THREAD

**Issue by muscleman** [MoneroOcean/node-cryptonight-hashing#22]
1) get compilation errors
2) readme states. cryptonight-pico, when it's really cryptonight_pico

**Comment by monerador** [comment]
same here:

> cryptonight-hashing@6.0.0 install /home/x/pool/node_modules/cryptonight-hashing
> node-gyp rebuild

make: Entering directory '/home/x/pool/node_modules/cryptonight-hashing/build'
  CC(target) Release/obj.target/cryptonight-hashing/xmrig/crypto/asm/cn_main_loop.o
  CXX(target) Release/obj.target/cryptonight-hashing/xmrig/crypto/cn_gpu_avx.o
../xmrig/crypto/cn_gpu_avx.cpp: In function ‘__m256 fma_break(const __m256&)’:
../xmrig/crypto/cn_gpu_avx.cpp:40:40: warning: AVX vector return without AVX enabled changes the ABI [-Wpsabi]
 inline __m256 fma_break(const __m256& x) 
                                        ^
In file included from /usr/lib/gcc/x86_64-linux-gnu/5/include/immintrin.h:41:0,
                 from /usr/lib/gcc/x86_64-linux-gnu/5/include/x86intrin.h:46,
                 from ../xmrig/crypto/cn_gpu_avx.cpp:28:
/usr/lib/gcc/x86_64-linux-gnu/5/include/avxintrin.h: In function ‘void prep_dv_avx(__m256i*, __m256i&, __m256&)’:
/usr/lib/gcc/x86_64-linux-gnu/5/include/avxintrin.h:442:1: error: inlining failed in call to always_inline ‘__m256 _mm256_cvtepi32_ps(__m256i)’: target specific option mismatch
 _mm256_cvtepi32_ps (__m256i __A)
 ^
../xmrig/crypto/cn_gpu_avx.cpp:37:29: error: called from here
     n01 = _mm256_cvtepi32_ps(v);
                             ^
In file included from /usr/lib/gcc/x86_64-linux-gnu/5/include/immintrin.h:41:0,
                 from /usr/lib/gcc/x86_64-linux-gnu/5/include/x86intrin.h:46,
                 from ../xmrig/crypto/cn_gpu_avx.cpp:28:
/usr/lib/gcc/x86_64-linux-gnu/5/include/avxintrin.h:882:1: error: inlining failed in call to always_inline ‘__m256i _mm256_load_si256(const __m256i*)’: target specific option mismatch
 _mm256_load_si256 (__m256i const *__P)
 ^
../xmrig/crypto/cn_gpu_avx.cpp:36:26: error: called from here
     v = _mm256_load_si256(idx);
                          ^
cryptonight-hashing.target.mk:138: recipe for target 'Release/obj.target/cryptonight-hashing/xmrig/crypto/cn_gpu_avx.o' failed
make: *** [Release/obj.target/cryptonight-hashing/xmrig/crypto/cn_gpu_avx.o] Error 1
\

**Comment by MoneroOcean** [comment]
Yes, this is currently expected. I used developer version of xmrig and cn/gpu is not working fine here. Will put more time later to investigate this. Can suggest to use tagged versions at this moment if you need stable release.

**Comment by muscleman** [comment]
yeah, i went back to this tag.

"cryptonight-hashing": "git://github.com/MoneroOcean/node-cryptonight-hashing.git#e0704bd7e9e88f9bf8d8d5fe54766915b39b138b",

**Comment by MoneroOcean** [comment]
Can you please check it now?

**Comment by synergyits** [comment]
Hi folks, just to let you know I too am having the same difficulty as **[monerador](/monerador) ** comments. The tag version that is most recent that will compile is v5.02 (Im on Ubuntu 16.4 and Node.js 10.0.0 (or 11.9.0 as well))
